cf2/Asset/Register.php

```diff
@@ -0,0 +1,199 @@
+<?php
+
+
+namespace calderawp\calderaforms\cf2\Asset;
+
+
+class Register
+{
+    /**
+     *
+     * @since 1.9.0
+     *
+     * @var string
+     */
+
+    protected $handle;
+    /**
+     *
+     * @since 1.9.0
+     *
+     * @var array
+     */
+    protected $data;
+
+    /**
+     *
+     * @since 1.9.0
+     *
+     * @var string
+     */
+    protected $scriptUrl;
+
+    /**
+     *
+     * @since 1.9.0
+     *
+     * @var string
+     */
+    protected $assetsFilePath;
+
+    /**
+     *
+     *
+     * @since 1.9.0
+     *
+     * @var bool
+     */
+    protected $registered;
+
+    /**
+     * Register constructor.
+     *
+     * @param $handle
+     * @param $coreUrl
+     * @param $corePath
+     * @param array $data
+     *
+     * @since 1.9.0
+     *
+     */
+    public function __construct($handle, $coreUrl, $corePath, array $data = [])
+    {
+        $this->handle = $handle;
+        $this->data = $data;
+        $this->registered = false;
+        $this->setScriptUrl($coreUrl . 'clients/' . $this->handle . '/build/index.min.js');
+        $this->setAssetsFilePath($corePath . '/clients/' . $this->handle . '/build/index.min.asset.json');
```

Comment:
From my logs $corePath seems to be set with an ending slash, clients should be set without first slash as on above line ( 'clients/' )

Reply:
>From my logs $corePath seems to be set with an ending slash, clients should be set without first slash as on above line ( 'clients/' )

Correct

 I fixed that here: https://github.com/CalderaWP/Caldera-Forms/pull/3519/commits/d32dbb84a7b2fbe965c3d6feae1e414b2d9ec0ff#diff-f15134c609c4cc7436fa7e92019ad12fR67